state/state.go

```diff
@@ -672,6 +672,11 @@ func (ls *LedgerState) Tip() ochainsync.Tip {
 	return ls.currentTip
 }
 
+// GetCurrentPParams returns the currentPParams value
+func (ls *LedgerState) GetCurrentPParams() any {
```

Comment:
Change the return type from `any` to `lcommon.ProtocolParameters` and then you can use `.Utxorpc()` on the return value directly

Reply:
I had originally done that but didn't type cast it, so I stuck w/ any and left it to downstream. This is way better and what I originally intended. Thanks.